public/config/configuration.json

```diff
@@ -2,10 +2,10 @@
   "BUSINESSES_URL": "https://dev.bcregistry.ca/business/auth/",
   "AUTH_URL": "https://dev.bcregistry.ca/business/auth/",
   "DASHBOARD_URL": "https://dev.bcregistry.ca/business/",
-  "LEGAL_API_URL": "https://legal-api-dev.pathfinder.gov.bc.ca/api/v1/",
-  "AUTH_API_URL": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1/",
-  "SBC_CONFIG_AUTH_API_URL": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1",
-  "PAY_API_URL": "https://pay-api-dev.pathfinder.gov.bc.ca/api/v1/",
+  "LEGAL_API_URL": "https://legal-api-dev.apps.silver.devops.gov.bc.ca/api/v1/",
+  "AUTH_API_URL": "https://auth-api-dev.apps.silver.devops.gov.bc.ca/api/v1/",
+  "SBC_CONFIG_AUTH_API_URL": "https://auth-api-dev.apps.silver.devops.gov.bc.ca/api/v1",
+  "PAY_API_URL": "https://pay-api-dev.apps.silver.devops.gov.bc.ca/api/v1/",
```

Comment:
Looks fine but I already updated these, causing you a conflict (sorry). I did text you in RC to see what project(s) you were updating but didn't see a response so I went ahead and updated all projects.

Reply:
I resolved the conflict